internal/benchrunner/runners/rally/runner.go

```diff
@@ -274,6 +341,43 @@ func (r *runner) setUp() error {
 	return nil
 }
 
+func (r *runner) extractSimulatedTemplate(indexTemplate string) (string, error) {
+	simulateTemplate, err := r.options.ESAPI.Indices.SimulateTemplate(r.options.ESAPI.Indices.SimulateTemplate.WithName(indexTemplate))
+	if err != nil {
+		return "", fmt.Errorf("error simulating template from composable template: %s: %w", indexTemplate, err)
+	}
+
+	templateBody, err := io.ReadAll(simulateTemplate.Body)
+	if err != nil {
+		return "", fmt.Errorf("error reading simulated template from composable template: %s: %w", indexTemplate, err)
+	}
+
+	var simulatedTemplate map[string]interface{}
+	err = json.Unmarshal(templateBody, &simulatedTemplate)
+	if err != nil {
+		return "", fmt.Errorf("error unmarshaling simulated template from composable template: %s: %w", indexTemplate, err)
+	}
+
+	simulatedTemplate["priority"] = 1000
+	simulatedTemplate["index_patterns"] = []string{indexTemplate + "-ep"}
+
+	indexTimeSeries := map[string]interface{}{
+		"start_time": "2000-01-01T00:00:00Z",
+		"end_time":   "2099-12-31T23:59:59Z",
+	}
+
+	simulatedTemplate["template"].(map[string]interface{})["settings"].(map[string]interface{})["index"].(map[string]interface{})["time_series"] = indexTimeSeries
+
+	delete(simulatedTemplate, "overlapping")
+
+	newTemplate, err := json.Marshal(simulatedTemplate)
```

Comment:
Nit. This will encode html entities, please check if this can be a problem here. If it is an issue here you can use a formatter like this one https://github.com/elastic/elastic-package/blob/main/internal/formatter/json_formatter.go#L65.